sass/directives/_boxes.scss

```diff
@@ -51,12 +51,19 @@
   position: relative;
   margin: $base-spacing-small;
   text-decoration: none;
+  font-weight: $font-weight-normal;
+
+  &:hover {
```

Comment:
@arelia unfortunately, there's no way to re-introduce the active/focus state styles without having them be triggered in unexpected ways. We'll have to explore alternatives in a defect. In the meantime, I will re-introduce the states and note in the Trello card, PR and JIRA ticket that their will be unexpected behavior on this page. Thanks for bringing this up!